env_run.xml

```diff
@@ -252,7 +252,7 @@
       timing tests.
     </desc>
     </entry>
-    <entry id="JOB_IDS" value="case.st_archive:1350468.chadmin1.ib0.cheyenne.ucar.edu">
+    <entry id="JOB_IDS" value="case.run:1366753.chadmin1.ib0.cheyenne.ucar.edu, case.st_archive:1366754.chadmin1.ib0.cheyenne.ucar.edu">
```

Comment:
I don't understand why there are two jobs listed here - there has never been two before.

Reply:
That's because I decided to tell st_archive to run upon the successful completion
of the assimilation.  When I was running short jobs that was poor choice,
but it speeds up the process when I'm running a whole month at a time.

Reply:
I guess it's a bit of a puzzle why the old version only listed st_archive.

Reply:
did the question "how does it know the assimilation finished ok before kicking off the archiver?" get answered?

Reply:
kevin answered this - see comment at the end of the pull request.  (kevin - you did reply to github so the answer is available here - that's all i wanted to ensure!)